### PR TITLE
add helm chart

### PR DESCRIPTION
**Pull request by dmitry231187**
add helm chart and CD
test url https://api.bpzb.smodata.net/authz/api/v1/authz/health_check

### CODE REVIEW

File: charts/authz/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+              protocol: {{ .Values.nginx.ports.protocol }}
+          volumeMounts:
+            - name: {{ .Values.nginx.volumeMounts.name }}
+              mountPath: {{ .Values.nginx.volumeMounts.mountPath }}
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          #ports:
```
Comment (ggramal) [inline charts/authz/templates/deployment.yaml]: we need to have liveness and readiness probes

File: charts/authz/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "authz.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
```
Comment (ggramal) [inline charts/authz/templates/deployment.yaml:38]: we also need a way to set env variables

File: charts/authz/templates/deployment.yaml (hunk 1 above)
Comment (dmitry231187) [reply charts/authz/templates/deployment.yaml]: ok. done

File: charts/authz/templates/deployment.yaml (hunk 2 above)
Comment (dmitry231187) [reply charts/authz/templates/deployment.yaml:38]: ok. done

**Review by ggramal — APPROVED** [review]

**Pull request merged**